root/static/scripts/release-editor/actions.js

```diff
@@ -69,6 +75,83 @@ const actions = {
     }));
   },
 
+  duplicateReleaseGroups: ko.observableArray([]),
+  loadingDuplicateReleaseGroups: ko.observable(false),
+  failedLoadingDuplicateReleaseGroups: ko.observable(false),
+  duplicateReleaseGroupsRequest: null,
+  duplicateReleaseGroupsQuery: null,
+
+  /**
+   * Query for existing release groups that are credited to any of the
+   * currently-credited artists and have similar names to the release.
+   */
+  loadDuplicateReleaseGroups() {
+    const release = this.rootField.release.peek();
+    const query = utils.constructLuceneFieldConjunction({
+      arid: release.artistCredit().names
+        .map((a) => a.artist?.gid)
+        .filter(Boolean),
+      releasegroup: [utils.escapeLuceneValue(release.name() ?? '')],
+    });
+    if (query === this.duplicateReleaseGroupsQuery) {
+      return;
+    }
+
+    // Cancel any in-progress lookup and clear existing results.
+    this.duplicateReleaseGroupsRequest?.abort();
+    this.duplicateReleaseGroupsRequest = null;
+    this.loadingDuplicateReleaseGroups(false);
+    this.failedLoadingDuplicateReleaseGroups(false);
+    this.duplicateReleaseGroupsQuery = query;
+    this.duplicateReleaseGroups.removeAll();
+
+    /*
+     * Make sure that an existing release group isn't selected
+     * and that there's a title and artist to use for searching.
+     */
+    if (
+      release.releaseGroup().gid ||
+      (release.name() ?? '') === '' ||
+      !release.artistCredit().names.some((a) => a.artist?.gid)
+    ) {
+      return;
+    }
+
+    this.loadingDuplicateReleaseGroups(true);
+    this.duplicateReleaseGroupsRequest = request({
+      url: '/ws/js/release-group' +
+        '?direct=false' +
+        '&advanced=true' +
+        '&limit=' + encodeURIComponent(maxDuplicateReleaseGroups) +
+        '&q=' + encodeURIComponent(query),
+    })
+      .always(() => {
```

Comment:
I had to change the indenting here to avoid what seemed like it might be an ESLint bug:

```
  136:7  warning  Expected line break before `.done`  @stylistic/js/newline-per-chained-call
```

I think the indenting I was using that triggered the warning was like:

```js
this.duplicateReleaseGroupsRequest = request({
  // ...
}).always(() => {
  // ...
}).fail(() => {
  // ...
}).done((data) => {
  // ...
});
```